src/vm/gcenv.ee.cpp

```diff
@@ -1349,3 +1349,10 @@ void GCToEEInterface::HandleFatalError(unsigned int exitCode)
 {
     EEPOLICY_HANDLE_FATAL_ERROR(exitCode);
 }
+
+bool GCToEEInterface::ShouldFinalizeObjectForUnload(AppDomain* pDomain, Object* obj)
+{
+    // CoreCLR does not have appdomains, so this code path is dead. Other runtimes may
+    // choose to inspect the object being finalized here.
+    return true;
```

Comment:
>  return true; [](start = 2, length = 14)

For changes like this that are very specific to core, you might want to have a uniformed way to annotate them so it'll be easy for you to know to take care of them when we need to port these back to desktop. Perhaps mark with [desktop todo] or something like that.